src/components/app.tsx

```diff
@@ -191,15 +192,19 @@ export class App extends React.PureComponent<IProps, IState> {
             projectId={activity.project_id}
           />
         }
-        { (activity.layout === ActivityLayouts.SinglePage || (currentPage !== 0 && !activity.pages[currentPage - 1].is_completion)) &&
+        { (activity.layout === ActivityLayouts.SinglePage || !isCompletionPage) &&
           <ExpandableContainer
             activity={activity}
             pageNumber={currentPage}
             page={activity.pages.filter((page) => !page.is_hidden)[currentPage - 1]}
             teacherEditionMode={teacherEditionMode}
             pluginsLoaded={pluginsLoaded}
+            glossaryPlugin={glossaryEmbeddable !== undefined}
```

Comment:
I would use `!= null` or even just `!!glossaryEmbeddable` here on the grounds that `null` or other falsy values aren't valid either.